external/kafka-0-10/src/test/scala/org/apache/spark/streaming/kafka010/DirectKafkaStreamSuite.scala

```diff
@@ -547,20 +559,20 @@ class DirectKafkaStreamSuite
     val ekp = new JHashMap[String, Object](kafkaParams)
     KafkaUtils.fixKafkaParams(ekp)
 
-    val s = new DirectKafkaInputDStream[String, String](
-      ssc,
-      preferredHosts,
-      new ConsumerStrategy[String, String] {
-        def executorKafkaParams = ekp
-        def onStart(currentOffsets: Map[TopicPartition, Long]): Consumer[String, String] = {
-          val consumer = new KafkaConsumer[String, String](kafkaParams)
-          val tps = List(new TopicPartition(topic, 0), new TopicPartition(topic, 1))
-          consumer.assign(Arrays.asList(tps: _*))
-          tps.foreach(tp => consumer.seek(tp, 0))
-          consumer
-        }
+    val newStrategy = new ConsumerStrategy[String, String] {
```

Comment:
Yeah. this one was a style thing. the code looked too nested to be easily readable.